### PR TITLE
feat(issues) Add waiting/sample state to org issue list

### PR DESCRIPTION
**Pull request by markstory**
Add the ErrorRobot waiting state to the org issue list. I added tests for ErrorRobot as there were none before.

screen shot 2019-01-22 at 3 15 43 pm

Fixes APP-993

### CODE REVIEW

File: tests/js/spec/components/errorRobot.spec.jsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import {browserHistory} from 'react-router';
+import {shallow} from 'enzyme';
+import {Client} from 'app/api';
+import ErrorRobot from 'app/components/errorRobot';
+
+jest.mock('react-router', () => {
```
Comment (billyvg) [inline tests/js/spec/components/errorRobot.spec.jsx]: This is globally mocked in `tests/js/setup.js`

File: src/sentry/static/sentry/app/views/organizationStream/overview.jsx [hunk 2]
```diff
@@ -509,6 +528,21 @@ const OrganizationStream = createReactClass({
     });
   },
 
+  renderAwaitingEvents(projects) {
+    let {organization} = this.props;
+    let project = projects.length === 1 ? projects[0] : null;
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/organizationStream/overview.jsx]: So if "all projects" are selected we pass `null`, what does `ErrorRobot` do when `project` is `null`?

Is using the first project the pattern we're using for now?

Comment (markstory) [reply src/sentry/static/sentry/app/views/organizationStream/overview.jsx]: I wasn't able to get the selected projects to be `null`. I always got either an empty array or one populated with the selected projects.

The organization issue list gets pushed the current selection through its props. It should inherit the first project pattern that we're using. This code is aiming to guard against the situation where an org has access to multi-project selection and selects multiple projects with no events.

**Review by billyvg — APPROVED** [review]
🤖

**Pull request merged**